remoteappmanager/docker/container.py

```diff
@@ -32,12 +32,22 @@ class Container(HasTraits):
     #: the id that will go in the URL of the container
     url_id = Unicode()
 
+    #: The absolute base prefix path for the container.
+    base_urlpath = Unicode(None, allow_none=True)
+
     @property
-    def url(self):
+    def urlpath(self):
         """Returns the relative url of the Container."""
         return "containers/{}".format(self.url_id)
 
     @property
+    def absurlpath(self):
```

Comment:
Actually, I think that we should use urlpath() for the absolute path and that's it, but again, I am not sure of the direction of this interface.